woocommerce/woocommerce-gateway-stripe
# Feature: Disconnect account

Fixes #2005

### Description
The `Disconnect` button in the `Account Details` section shows a confirmation modal. After confirming to disconnect via modal all the live and test account keys are removed from DB. 

![diconnect](https://user-images.githubusercontent.com/33387139/136517946-650f588a-eaad-4d4a-b5eb-f518fd1fb333.gif)


### Testing instructions
- Make sure you have test keys or live keys set
- Enable `_wcstripe_feature_upe_settings`
- Go to the `Settings` tab in `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe`
- Click `Disconnect` from the 3 dot menu of the `Account Details` section and the modal will appear

### QA steps
- [ ]  Click the `Cancel` or `Cross` icon on the modal -> Modal disappears and nothing changes, previous keys are still present in DB
- [ ] Click `Disconnect` on the modal -> Modal disappears, the window reloads the `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe` page and the "connect stripe account" card is now there on the page.
- [ ] All the test and live keys are empty in the DB
